packages/skia/cpp/api/JsiSkImage.h

```diff
@@ -271,7 +275,22 @@ class JsiSkImage : public JsiSkWrappingSkPtrHostObject<SkImage> {
   JsiSkImage(std::shared_ptr<RNSkPlatformContext> context,
              const sk_sp<SkImage> image)
       : JsiSkWrappingSkPtrHostObject<SkImage>(std::move(context),
-                                              std::move(image)) {}
+                                              std::move(image)),
+        _deletionHandler() {
```

Comment:
looks a bit weird to have empty constructor on initialization list. This is the default one, so the _deletionHandler will be initialized with it even if its not here/

Reply:
yes that's a nice catch thank you.